clang/test/CodeGen/cheri/cheri-debug-info.cpp

```diff
@@ -19,15 +22,15 @@ char *__capability always_capability_pointer;
 char &__capability always_capability_reference = *always_capability_pointer;
 char &&__capability always_capability_rvalue_reference = 0;
 
-// CHECK-HYBRID-DAG: ![[DWARF_SIMPLE_POINTER:[0-9]+]] = !DIDerivedType(tag: DW_TAG_pointer_type, baseType: !{{[0-9]+}}, size: 64)
-// CHECK-HYBRID-DAG: ![[DWARF_SIMPLE_REFERENCE:[0-9]+]] = !DIDerivedType(tag: DW_TAG_reference_type, baseType: !{{[0-9]+}}, size: 64)
-// CHECK-HYBRID-DAG: ![[DWARF_SIMPLE_RVALUE_REFERENCE:[0-9]+]] = !DIDerivedType(tag: DW_TAG_rvalue_reference_type, baseType: !{{[0-9]+}}, size: 64)
+// CHECK-HYBRID-DAG: ![[DWARF_SIMPLE_POINTER:[0-9]+]] = !DIDerivedType(tag: DW_TAG_pointer_type, baseType: !{{[0-9]+}}, size: [[#POINTER_SIZE]])
+// CHECK-HYBRID-DAG: ![[DWARF_SIMPLE_REFERENCE:[0-9]+]] = !DIDerivedType(tag: DW_TAG_reference_type, baseType: !{{[0-9]+}}, size: [[#POINTER_SIZE]])
+// CHECK-HYBRID-DAG: ![[DWARF_SIMPLE_RVALUE_REFERENCE:[0-9]+]] = !DIDerivedType(tag: DW_TAG_rvalue_reference_type, baseType: !{{[0-9]+}}, size: [[#POINTER_SIZE]])
 // CHECK-128-DAG: ![[DWARF_CAPABILITY_POINTER:[0-9]+]] = !DIDerivedType(tag: DW_TAG_pointer_type, baseType: !{{[0-9]+}}, size: 128)
```

Comment:
These can be combined too

Reply:
Yes, but then I need to add a new filecheck -D flag to all 6 run lines.